forum/models/threads.py

```diff
@@ -42,19 +44,7 @@ def get_votes(self, up=None, down=None):
         }
         return votes
 
-    def insert(
```

Comment:
Yeah, I don't think we should remove these arguments here. Does the issue come from the non-named args? I believe we can address that by modifying the signature of the insert method from the base class and setting it to `def insert(*args, **kwargs)`. (note the additional `*args`)

Reply:
i'll try it.

Reply:
By using args in base class, we can use arguments, but we'll get these issues
```
forum/models/threads.py:147: error: Incompatible types in assignment (expression has type "Literal[True]", target has type "str")  [assignment]
forum/models/threads.py:149: error: Incompatible types in assignment (expression has type "Literal[True]", target has type "str")  [assignment]
forum/models/threads.py:153: error: Incompatible types in assignment (expression has type "list[str]", target has type "str")  [assignment]
forum/models/threads.py:155: error: Incompatible types in assignment (expression has type "Literal[True]", target has type "str")  [assignment]
forum/models/threads.py:163: error: Incompatible types in assignment (expression has type "dict[str, int]", target has type "str")  [assignment]
forum/models/threads.py:165: error: Incompatible types in assignment (expression has type "list[str]", target has type "str")  [assignment]
forum/models/threads.py:169: error: Incompatible types in assignment (expression has type "Literal[True]", target has type "str")  [assignment]
forum/models/threads.py:171: error: Incompatible types in assignment (expression has type "int", target has type "str")  [assignment]
forum/models/threads.py:173: error: Incompatible types in assignment (expression has type "Literal[True]", target has type "str")  [assignment]
forum/models/threads.py:176: error: Incompatible types in assignment (expression has type "datetime", target has type "str")  [assignment]
forum/models/threads.py:177: error: Incompatible types in assignment (expression has type "datetime", target has type "str")  [assignment]
```
It can be resolved by using Any instead of other type(other than str).